### ISSUE TITLE
Localization Translate not working on iOS

### ISSUE THREAD

**Issue by JulioRosil** [Jumio/mobile-react#42]
I'm trying to translate the labels text following the .GIF you posted on this documentation.
I have added the files for Spanish in Xcode, but in the app it still appears on English.

Working with Xcode v14.1, react-native v0.68.2, react v17.0.2, Jumio v4.3.1

Post reference images.

![Captura 1](https://user-images.githubusercontent.com/99291594/200028026-9c740d7e-6af3-45d9-979b-700c240e126c.png)
![captura 2](https://user-images.githubusercontent.com/99291594/200028110-8aa77143-5af4-4fac-971c-9627d9e4d819.png)

**Comment by BernadetteTh** [comment]
Hello @JulioRosil 
just looking at these localizable file names, this looks like you are using very outdated Localization files if you are on plugin 4.3.1. 

After running `pod install` there should be a Localization folder in your Pods directory. Please add the language you need from there.

**Comment by JulioRosil** [comment]
Hello @BernadetteTh 
You are correct, we copied that files from an older version of Jumio, because we have another project with version 3.9.0 and the Localization works.

We tried adding just the new file in Pods, but it doesn't work too, you can see the following image, with this unique file it also shows on English, Spanish never works
![captura 3](https://user-images.githubusercontent.com/99291594/200054521-ff837fa2-a02f-41c8-ad0b-c72537f55e1c.png)

**Comment by BernadetteTh** [comment]
@JulioRosil please add English localizable as well. 
From the screenshot, it looks like this hasn't happened yet - no English or Base Localization.

**Comment by JulioRosil** [comment]
Hello @BernadetteTh ,

I've added too the English language, I have Spanish, Spanish Latin and English, and it still shows in English. We really need to show exclusively on Spanish, better if its LatinAmerican.

![captura 4](https://user-images.githubusercontent.com/99291594/200341026-c82fcac0-017a-4d9b-a2bc-3b667b977c58.png)

**Comment by BernadetteTh** [comment]
@JulioRosil 
Please make sure to only add Localizable string files to your project that are provided in the downloaded pods, in this case `es.proj` for generalized Spanish, instead of localized Latin American Spanish.

**Comment by BernadetteTh** [comment]
Hello @JulioRosil 
I'm assuming you've cleaned the Xcode project, reset the React Native cache and re-built the app?

**Comment by JulioRosil** [comment]
Hello @BernadetteTh 
Yes i have. Cleaned the project, rebuilted, pod deintegrate, pod install, pod update, clean derived data, close the project, turn off the mac... all of them

**Comment by BernadetteTh** [comment]
@JulioRosil 
please make sure Localizable resources are included in **"Copy Bundle Resources"** step of your **build phases**:  
![Screenshot 2022-11-09 at 16 09 06](https://user-images.githubusercontent.com/28044295/200866795-d39cd620-4067-4fac-87eb-2730a070bba0.png)

If that is already the case, please walk us through your process of including the Jumio plugin in your project step by step.
Please also check if the same issue happens with our DemoApp using the same device.

**Comment by JulioRosil** [comment]
Hello @BernadetteTh 
Yes, I have added that files in Copy Bundle Resources.

I can make you a walkthrough:

First we installed Jumio Plugin with NPM
![image](https://user-images.githubusercontent.com/99291594/201160228-907f56d8-c74e-47dd-abd2-949e78b522a3.png)


Then, we set Camera permission on Info.plist
![image](https://user-images.githubusercontent.com/99291594/201160426-3c38ac93-4946-4bb5-aa93-259d264b02ba.png)

We added this on the Pod file, and made Pod install
![image](https://user-images.githubusercontent.com/99291594/201160778-262b8d1a-54f0-4988-b6cb-a61df745e3b4.png)

We can't add the Overriding the static_framework Pre_install in pods, because it makes conflicts with another libraries we have.

And that's all how we use Jumio plugin in our app.

**Comment by BernadetteTh** [comment]
Hello @JulioRosil 
just to clarify, were the Localizable resources already included in the **"Copy Bundle Resources"** section or did you have to add them?

**Comment by JulioRosil** [comment]
Hello @BernadetteTh 
There was already included, additionally I included manually too making. So we now have two instances.
But manually or automatically it doesn't worked, with 1 or 2 instances added.

![image](https://user-images.githubusercontent.com/99291594/201373309-871dd77e-6988-4142-9b3c-ebb5c07352e3.png)
![image](https://user-images.githubusercontent.com/99291594/201373351-25073a6a-ead7-45b7-a92c-e45125ea0c40.png)

**Comment by JulioRosil** [comment]
Hello @BernadetteTh and Jumio Team, 
We are trying to install and run the Demo App as we talk at the meeting this morning.
we just make `npm install` and `cd ios && pod install`, then we created the Provisioning Profile in Apple Developer portal, then we added the new Bundle Identifier to the project and Run.
The app runs and installs, but it shows a black space, we don't know how to proceed on this.
Appreciate your help.

![image](https://user-images.githubusercontent.com/99291594/202243133-c6da5d88-ac8c-4f5e-ac40-287a1569b8b4.png)
